src/enhancer.js

```diff
@@ -63,6 +64,7 @@ export default function enhanceWithRadium(
 
       this.state = this.state || {};
       this.state._radiumStyleState = {};
+      this.state._styleID = uuid.v4();
```

Comment:
Do we want this to be per-instance or per-component? #625 would be solved by per-component, but #593 it seems would not be.

If it is per-instance, you should move it directly to `this`, and prefix it with radium, just like `_radiumIsMounted` below.

If it is per component class, you can make it a static, like `_isRadiumEnhanced` above.